### DISCUSSION TITLE
SwitchHosts v4.0

### DISCUSSION THREAD

**Discussion by oldj** [oldj/SwitchHosts#561] [General]
After three weeks of beta testing, SwitchHosts v4.0 is finally officially released. 🎉

Compared with v3.x, this is a completely rewritten version, which establishes a better foundation for subsequent updates.

In terms of core functions (management, switching hosts), it has not changed much, but some minor issues that have been kept in the issue for a long time have been dealt with, such as using a proxy to update remote hosts, etc., and some experience details have also been obtained optimization.

If you find a problem or bug, please submit an issue feedback or PR!

Happy switching!

---

经过三周的 beta 测试，SwitchHosts v4.0 终于正式发布了。🎉

与 v3.x 相比，这是一个彻底重写的版本，为后续的更新建立了更好的基础。

在核心功能（管理、切换 hosts）上，它没有太大的变化，不过一些在 issue 中保持了很长时间的小问题得到了处理，比如使用代理更新远程 hosts 等等，一些体验细节也得到了优化。

如果发现问题或 bug，欢迎提交 issue 反馈或者 PR！

祝 Switch 愉快！

<hr /><em>This discussion was created from the release <a href='https://github.com/oldj/SwitchHosts/releases/tag/v4.0.0'>SwitchHosts v4.0</a>.</em>

**Comment by monocor** [comment]
Hi, is there a way to switch the interface language from Chinese to English (my Mac language setting is German but I see Chinese texts only)? I'd really like to use this app but currently I can't because I don't speak Chinese 😅 .

**Reply by oldj** [reply]
Sorry, it was my negligence, the language judgment at startup may need to be improved.
You can change the language in the preferences panel, I made an illustration, hope this will help you:

![change_language](https://user-images.githubusercontent.com/416130/115488000-60e9fa80-a28c-11eb-9374-ac867fb3eee3.png)

**Reply by oldj** [reply]
There is one last step, don’t forget to click the "OK"(确定) button in the lower right corner. 😅

![image](https://user-images.githubusercontent.com/416130/115489384-b6270b80-a28e-11eb-8b3b-5466b7373585.png)

**Reply by pitichampi** [reply]
Many thanks ! that is useful indeed ^^

**Comment by ElfSundae** [comment]
I just updated the cask spec for SwitchHosts 4.x: https://github.com/Homebrew/homebrew-cask/commit/93620f6b0585a30df0d0fd0550f9f754cee85bf4
```sh
brew install switchhosts
```

**Reply by oldthreefeng** [reply]
> brew install switchhosts

@ElfSundae  still install version : v3.5.6/SwitchHosts._macOS_3.5.6.5551.dmg
